sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -178,21 +190,22 @@ public AbstractScheduler setOfferQueueSize(int queueSize) {
      */
     @VisibleForTesting
     public Collection<Plan> getPlans() {
-        return mesosScheduler.planCoordinator.getPlanManagers().stream()
+        return getPlanCoordinator().getPlanManagers().stream()
                 .map(planManager -> planManager.getPlan())
                 .collect(Collectors.toList());
     }
 
     /**
      * Returns a list of API resources to be served by the scheduler to the local cluster.
-     * This may be called before {@link #getPlanCoordinator()} has been called.
      */
     public abstract Collection<Object> getResources();
 
     /**
-     * Returns the {@link PlanCoordinator}.
+     * Provides a callback to indicate that the scheduler has registered with Mesos.
      */
-    protected abstract PlanCoordinator getPlanCoordinator() throws Exception;
+    protected abstract PlanCoordinator getPlanCoordinator();
+
+    protected abstract void registeredWithMesos();
```

Comment:
Does anything call this? I see declarations, but no calls

Reply:
Thanks.  Good catch.  I lost that.